src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$size-day: 100px;
```

Comment:
For variable names, it is better to give more precise names, for example: `$cell-size`, `$day-cell-size`.
This applies to all variables.